applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -629,12 +657,12 @@ namespace Kratos
 
 
 		VectorNorm = MathUtils<double>::Norm(deltaX);
-		if (VectorNorm != 0.00) deltaX /= VectorNorm;
+		if (VectorNorm > numerical_limit) deltaX /= VectorNorm;
 

```

Comment:
Here  i would throw an error when the norm is small and KRATOS_DEBUG is defined

Reply:
Can you give me an example on how to check if KRATOS_DEBUG is defined?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/deprecating_cross_products/kratos/utilities/math_utils.h#L858